docs/azure/sdk/protocol-convenience-methods.md

```diff
@@ -63,14 +63,14 @@ The preceding code demonstrates the following `Azure.Core` convenience method pa
 
 The following code uses a `ContentSafetyClient` to call the `AnalyzeText` protocol method:
 
-:::code source="snippets/protocol-convenience-methods/AzureCoreProtocol/Program.cs" highlight="18-23":::
+:::code source="snippets/protocol-convenience-methods/AzureCoreProtocol/Program.cs" highlight="19-24":::
 
 The preceding code demonstrates the following protocol method patterns:
 
 - Uses the `RequestContent` type to supply data for the request body.
 - Uses the `RequestContext` type to configure request options.
 - Returns data using the `Response` type.
-- Reads `response.Content` to access the response data.
+- Reads content from the response data into a [dynamic](../../csharp/advanced-topics/interop/using-type-dynamic.md) type using <xref:Azure.AzureCoreExtensions.ToDynamicFromJson%2A>.
```

Comment:
It might be preferable to link this blog post: https://devblogs.microsoft.com/azure-sdk/dynamic-json-in-azure-core/
It will give more context about the intended use and capabilities of the dynamic instance returned from `ToDynamicFromJson`.